Source/Core/VideoBackends/D3D/D3DTexture.cpp

```diff
@@ -11,95 +11,85 @@
 
 namespace DX11
 {
-D3DTexture2D* D3DTexture2D::Create(unsigned int width, unsigned int height, D3D11_BIND_FLAG bind,
-                                   D3D11_USAGE usage, DXGI_FORMAT fmt, unsigned int levels,
-                                   unsigned int slices, D3D11_SUBRESOURCE_DATA* data)
+void D3DTexture2D::Initialize(DXGI_FORMAT format, unsigned int width, unsigned int height,
```

Comment:
How are you supposed to know if the object created is valid or not?  This should probably return a bool?